libc/stdio_file.c

```diff
@@ -170,7 +186,11 @@ TEST(stdio_fopenfclose, fdopen_file)
 		filep2 = fdopen(fd, "r");
 		TEST_ASSERT_NOT_NULL(filep2);
 	}
-	TEST_ASSERT_EQUAL_INT(0, fclose(filep));
+	ret = fclose(filep);
```

Comment:
leaking `filep2` in this test

Reply:
this is a problem with the testcase, not with the coding style so it can be taken care of in separate PR if You would prefer (I'm eager to merge this PR)

Reply:
Ok, so let's merge it. We will look at this issue

Reply:
Here is the proposed fix: https://github.com/phoenix-rtos/phoenix-rtos-tests/pull/150/files